.github/workflows/build_part_of_image_all_platforms.yaml

```diff
@@ -0,0 +1,80 @@
+# Reusable Workflow for building the Docker Image.
+name: Build Part of Image (All Platforms)
```

Comment:
It seems clunky to have this and `build_part_of_image.yaml`. It might be the easiest way, but idk